app/uk/gov/hmrc/taxenrolmentassignmentfrontend/config/AppConfig.scala

```diff
@@ -18,12 +18,15 @@ package uk.gov.hmrc.taxenrolmentassignmentfrontend.config
 
 import javax.inject.{Inject, Singleton}
 import play.api.Configuration
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 
 @Singleton
-class AppConfig @Inject()
-  (
-    config: Configuration
-  ) {
-  val welshLanguageSupportEnabled: Boolean = config.getOptional[Boolean]("features.welsh-language-support").getOrElse(false)
+class AppConfig @Inject()(config: Configuration,
+                          val servicesConfig: ServicesConfig) {
+  val welshLanguageSupportEnabled: Boolean = config
+    .getOptional[Boolean]("features.welsh-language-support")
+    .getOrElse(false)
```

Comment:
(Optional change) I know you just reformatted this but we could remove the configuration being brought in and replace the above with:
`  servicesConfig.getConfBool("features.welsh-language-support", defBool = false)
`

Reply:
done